test/ragnarok/ManagedPortfolio.test.ts

```diff
@@ -214,6 +208,38 @@ describe('ManagedPortfolio', () => {
     })
   })
 
+  describe('maxSize',() => {
+    it('prevents deposit if total after deposit > maxSize', async () => {
+      await portfolio.setMaxSize(0)
+      return expect(depositIntoPortfolio(10, lender)).to.be.revertedWith('ManagedPortfolio: Portfolio is full')
+    })
+
+    it('allows deposit if total after deposit = maxSize', async () => {
+      await portfolio.setMaxSize(parseUSDC(100))
+      await expect(depositIntoPortfolio(100, lender)).to.not.be.reverted
+    })
+
+    it('allows multiple deposits until total after deposit > maxSize', async () => {
+      await portfolio.setMaxSize(parseUSDC(100))
+      await expect(depositIntoPortfolio(50, lender)).to.not.be.reverted
+      await expect(depositIntoPortfolio(50, lender2)).to.not.be.reverted
+      await expect(depositIntoPortfolio(50, lender)).to.be.revertedWith('ManagedPortfolio: Portfolio is full')
+    })
+
+    it('allows deposits after a loan is issued', async () => {
```

Comment:
This test name doesn't sound right to me: in this test a deposit after issuing a loan fails.